README.md

```diff
@@ -47,49 +47,50 @@ We are _following_ a **_battle-tested_ industry-standard** approach
 and applying it to our Elixir/Phoenix App. <br />
 We are using:
 
-+ **Advanced Encryption Standard** (**AES**) to encrypt sensitive data.
-  + **Galois/Counter Mode**
-for _symmetric_ key cryptographic block ciphers:
-https://en.wikipedia.org/wiki/Galois/Counter_Mode
-recommended many security and cryptography practitioners including
- [Matthew Green](https://blog.cryptographyengineering.com/),
- [Niels Ferguson](https://en.wikipedia.org/wiki/Niels_Ferguson)
- and [Bruce Schneier](https://www.schneier.com/blog/about/)
-  + "Under the hood" we are using Erlang's
-[crypto](http://erlang.org/doc/man/crypto.html) library
-_specifically_ AES with **256 bit keys** <br />
-(_the same as AWS or Google's KMS service_)
-see: http://erlang.org/doc/man/crypto.html#block_encrypt-4
-+ Password "hashing" using the **Argon2**
-key derivation function (KDF): https://en.wikipedia.org/wiki/Argon2 <br />
-_specifically_ the Elixir implementation of `argon2`
-written by David Whitlock: https://github.com/riverrun/argon2_elixir
-which in turn uses the **C** "_reference implementation_"
-as a "Git Submodule".
+* **Advanced Encryption Standard** (**AES**) to encrypt sensitive data.
+  * **Galois/Counter Mode**
+    for _symmetric_ key cryptographic block ciphers:
+    https://en.wikipedia.org/wiki/Galois/Counter_Mode
+    recommended by many security and cryptography practitioners including
+    [Matthew Green](https://blog.cryptographyengineering.com/),
+    [Niels Ferguson](https://en.wikipedia.org/wiki/Niels_Ferguson)
+    and [Bruce Schneier](https://www.schneier.com/blog/about/)
+  * "Under the hood" we are using Erlang's
```

Comment:
I _prefer_ to use the `+` for bullet-points because the `*` is often used fro _emphasis_ ...